### PR TITLE
[BREAKING CHANGE] Move visualizations to core

### PR DESCRIPTION
**Pull request by balajimanoharan**
[BREAKING CHANGE] Move visualizations to core

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "test": "tests"
```
Comment (colemanProjects) [inline packages/core/package.json]: do we need a major version bump?

File: packages/core/app/styles/navi-core/utils/mixins.less [hunk 2]
```diff
@@ -46,3 +46,42 @@
...
+    -webkit-flex-flow: column;
+}
+
+.align-right{
+    justify-content: flex-end;
+    text-align: right;
+}
+
+.config-container {
+    margin: 5px;
```
Comment (mattgraham1995) [inline packages/core/app/styles/navi-core/utils/mixins.less]: 🔤

File: packages/core/app/styles/navi-core/utils/mixins.less [hunk 3]
```diff
@@ -46,3 +46,42 @@
...
+    text-align: left;
+    padding: 10px;
+    font-family: @font-family-thin;
+}
+
+.config-section {
+    width: 100%;
+    box-sizing: border-box;
+    background-color: @navi-white;
+    resize: none;
```
Comment (mattgraham1995) [inline packages/core/app/styles/navi-core/utils/mixins.less]: 🔤

File: packages/core/blueprints/navi-core/index.js [hunk 4]
```diff
@@ -10,10 +10,30 @@ module.exports = {
...
+        { name: 'ember-radio-button',           target: '1.2.1' },
+        { name: 'ember-sortable',               target: '1.10.0' },
+        { name: 'ember-cli-string-helpers',     target: '1.8.1' },
+        { name: 'ember-tether',                 target: '0.4.1' },
+        { name: 'ember-toggle',                 target: '5.2.1' },
+        { name: 'ember-tooltips',               target: '2.9.2' },
+        { name: 'ember-truth-helpers',          target: '2.0.0' },
+        { name: 'navi',                         target: '^8.0.0' },
+        { name: 'navi-data',                    target: '^8.0.0' },
+        { name: 'navi-core',                    target: '^8.0.0' }
```
Comment (mattgraham1995) [inline packages/core/blueprints/navi-core/index.js]: should navi-core be in the blueprint for navi-core?

File: packages/core/tests/dummy/config/environment.js [hunk 5]
```diff
@@ -70,10 +73,6 @@ module.exports = function(environment) {
     ENV.APP.LOG_VIEW_LOOKUPS = false;
 
     ENV.APP.rootElement = '#ember-testing';
-
-    ENV['ember-cli-mirage'] = {
-      enabled: false
```
Comment (mattgraham1995) [inline packages/core/tests/dummy/config/environment.js]: 😆

File: packages/reports/addon/services/navi-visualizations.js [hunk 6]
```diff
@@ -48,7 +48,7 @@ export default Ember.Service.extend({
    */
   all() {
     // Find all visualizations registered in requirejs under the namespace "components/navi-visualizations"
-    let visualizationRegExp = new RegExp(`^(?:${config.modulePrefix}/)?navi-visualizations/manifests/([a-z-]*)$`),
+    let visualizationRegExp = new RegExp(`^(?:${config.modulePrefix}/)?manifests/([a-z-]*)$`),
```
Comment (mattgraham1995) [inline packages/reports/addon/services/navi-visualizations.js]: Are you sure we want to remove this? Aren't the visualizations still in `.../components/navi-visualizations/...`?

File: packages/reports/blueprints/navi/index.js [hunk 7]
```diff
@@ -22,9 +22,6 @@ module.exports = {
         { name: 'ember-cli-string-helpers',         target: '^1.6.0' },
         { name: 'ember-cp-validations',             target: '3.1.2' },
         { name: 'ember-data-model-fragments',       target: '2.11.5' },
-        { name: 'navi-core',                        target: '^8.0.0' },
```
Comment (mattgraham1995) [inline packages/reports/blueprints/navi/index.js]: Why are we removing core and data from the reports blueprint?

File: packages/core/blueprints/navi-core/index.js (hunk 4 above)
Comment (balajimanoharan) [reply packages/core/blueprints/navi-core/index.js]: Ya, my bad will get rid of them.

File: packages/reports/addon/services/navi-visualizations.js (hunk 6 above)
Comment (balajimanoharan) [reply packages/reports/addon/services/navi-visualizations.js]: The manifests are not.

File: packages/reports/blueprints/navi/index.js (hunk 7 above)
Comment (balajimanoharan) [reply packages/reports/blueprints/navi/index.js]: Because version `8.0.0` does not exist.

**Pull request merged**